tutorials/pio/include.mdx

```diff
@@ -0,0 +1,182 @@
+---
+custom_edit_url: null
+---
+
+import Image from '/src/components/Images.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Part 2: Include Luos to your project
+
+## 1. Include Luos lib in your project
+
+The plateformio.ini files in your project describes precompiler variables and library depencies of your project.
+
+In our case we want to use Luos as Library. Put in Luos and LuosHAL in lib_deps will download from PIO registry Luos library and will build and link Luos to your project.
+
+Fill the plateformio.ini of your project :
+
+<Tabs>
+<TabItem value="ST" label="ST">
+
+```c
+build_unflags = -Os
+build_flags =
+		-O1
+    -include node_config.h
+    -DUSE_HAL_DRIVER
+    -DUSE_FULL_LL_DRIVER
+    -DLUOSHAL=STM32F0
+lib_deps =
+    luos_engine
+debug_tool = stlink
+upload_protocol = stlink
+```
+
+LUOSHAL=STM32F0 → select Luos HAL for MCU Family (here we use STM32F072Rb so F0 family
+
```

Comment:
I'm not sure why it's `LUOSHAL` here and `DLUOSHAL` in the code. Can you please check and confirm?